src/themes/teams/components/Menu/menuVariables.ts

```diff
@@ -48,5 +55,10 @@ export default (siteVars: any): MenuVariables => {
     disabledColor: siteVars.gray06,
 
     lineHeightBase: siteVars.lineHeightMedium,
+
+    horizontalPaddingLeft: pxToRem(18),
+    horizontalPaddingRight: pxToRem(18),
+    horizontalPaddingTop: pxToRem(14),
+    horizontalPaddingBottom: pxToRem(14),
```

Comment:
> When four values are specified, the paddings apply to the top, right, bottom, and left in that order (clockwise).
> https://developer.mozilla.org/en-US/docs/Web/CSS/padding#Values

My vote is to keep a single value:

```suggestion
horizontalPadding: `${pxToRem(14)} ${pxToRem(18)} ${pxToRem(14)} ${pxToRem(18)}`
```